dev-util/xmake/Manifest

```diff
@@ -1,3 +1 @@
 DIST xmake-v2.5.3.tar.gz 3268619 BLAKE2B 7277cc621a3ddd837f8a6abaa8abf78d94c2a2a0aa787cb0aa5df5f4bf0eaee57d126ffd962da1ba1be3fabe28ac352ee4e2a61b4e2b7ef279d05401b763d67e SHA512 ee72b392da77f8e07a9d22573a67c9461a3bd8bd535b25acd4300b6e568f76cbf4f1afb42e33a1c95592a355f7798c80d786554bd6d828f0bd3027f1fc2397d0
```

Comment:
we can use xmake-v2.5.4.tar.gz, I have released it.

Reply:
> we can use xmake-v2.5.4.tar.gz, I have released it.

sure, will bump

Reply:
done

Reply:
Thank you, but I have some doubts, do ebuilds have to be maintained separately for each version?

Reply:
> Thank you, but I have some doubts, do ebuild files have to be maintained separately for each version?

yes, this is how they are updated. There is also a 9999 version which is called "live". The 9999 ebuilds are updated each time they are recompiled because the build process fetches the git (or other VCS) changes.

Reply:
Got it. Thanks.